src/Controller/AdminController.php

```diff
@@ -34,6 +34,7 @@ protected function isActionAllowed($actionName)
         switch ($actionName) {
             // autocomplete action is mapped to list action for access permissions
             case 'autocomplete':
+                break;
```

Comment:
This is not related to the PR, so please don't mix many purposes in the same PR.

And moreover, it is wrong.